data/org.freedesktop.portal.Flatpak.xml

```diff
@@ -339,6 +339,36 @@
       <arg type="a{sv}" name="options" direction="in"/>
     </method>
 
+    <!--
+        Install:
+        @parent_window: Identifier for the application window,
+          see <link linkend="parent_window">Common Conventions</link>
+        @ref: ref to install
+        @options: Vardict with optional further information
+
+        Asks to install one or more Flatpak refs.
```

Comment:
The arguments list it as "ref" and it's a single string. Is there an example on how to specify multiple refs using a single argument?